packages/uhk-web/src/app/directives/tooltip/tooltip.directive.ts

```diff
@@ -56,6 +60,12 @@ export class TooltipDirective implements AfterContentInit, OnChanges {
             innerStyle = `style="max-width: ${this.maxWidth}px;"`;
         }
 
+        console.log({
```

Comment:
Why do you want to log this?

Reply:
I just used it for ugly debugging. Now deleted